src/main/webapp/index.html

```diff
@@ -20,6 +20,23 @@ <h1>
     <input type="text" id="node-name" name="search-name" placeholder="Name"></input>
   </div>
   <button onclick="graph.generateGraph()">Get Graph</button>
+
+  <div>
+    <label>Find Node in Graph: </label>
+    <input type="text" id="node-search" placeholder="Node Name"></input>
+    <button id="search-button">Find Node</button>
+    <label id="node-error"></label>
+  </div>
+
+  <div>
+    <label>Find Token in Graph: </label>
```

Comment:
![image](https://user-images.githubusercontent.com/48307028/88431612-51eb2000-cdc0-11ea-80f1-c2aa94cbad4b.png)
when I find F by node and then click "Find Token" with b.js, nothing is highlighted and I don't get an error with "Token doesn't exist" either. Do you know what's happening?

Reply:
I am not sure and could not recreate it myself. What did you do to get that to happen?

Reply:
have you tried navigating forward / back a few times and then searching for different things on the same graph? if you can't it's fine it might be a rare error I guess